apps/src/lib/levelbuilder/rubrics/EvidenceDescriptions.jsx

```diff
@@ -16,27 +19,44 @@ export default function EvidenceDescriptions({isAiEnabled}) {
         </Heading6>
       </div>
       <EvidenceDescriptionsRow
-        isAiEnabled={isAiEnabled}
+        isAiEnabled={learningGoalData.aiEnabled}
         evidenceLabel={'Extensive Evidence'}
+        evidenceLevelData={
+          learningGoalData.learningGoalEvidenceLevelsAttributes[3]
+        }
+        updateLearningGoal={updateLearningGoal}
       />
       <EvidenceDescriptionsRow
-        isAiEnabled={isAiEnabled}
+        isAiEnabled={learningGoalData.aiEnabled}
         evidenceLabel={'Convincing Evidence'}
+        evidenceLevelData={
+          learningGoalData.learningGoalEvidenceLevelsAttributes[2]
+        }
+        updateLearningGoal={updateLearningGoal}
       />
       <EvidenceDescriptionsRow
-        isAiEnabled={isAiEnabled}
+        isAiEnabled={learningGoalData.aiEnabled}
         evidenceLabel={'Limited Evidence'}
+        evidenceLevelData={
+          learningGoalData.learningGoalEvidenceLevelsAttributes[1]
+        }
+        updateLearningGoal={updateLearningGoal}
       />
       <EvidenceDescriptionsRow
-        isAiEnabled={isAiEnabled}
+        isAiEnabled={learningGoalData.aiEnabled}
         evidenceLabel={'No Evidence'}
+        evidenceLevelData={
```

Comment:
Could you use the [shared constants](https://github.com/code-dot-org/code-dot-org/blob/3b10cf57c934b880e0209ef6c9bd1ac85b17a154/lib/cdo/shared_constants.rb#L601) here? Here's an example of how I used them in the past: https://github.com/code-dot-org/code-dot-org/blob/3b10cf57c934b880e0209ef6c9bd1ac85b17a154/apps/src/templates/rubrics/rubricHelpers.js#L5

Reply:
oh I wasn't at all clear about what I was suggesting here. I was actually wondering if this could look like:
```
evidenceLabel={UNDERSTANDING_LEVEL_STRINGS[RubricUnderstandingLevel.NONE]}
evidenceLevelData={learningGoalData.learningGoalEvidenceLevelsAttrtibutes[RubricUnderstandingLevels.NONE]}
```

I realize now that's a bit verbose but it would be nice to avoid the index magic numbers. Up to you here -- I don't think this blocks merging!